libs/blocks/quiz/quiz.css

```diff
@@ -1,980 +1,384 @@
-.quiz { 
-  min-height: 1000px;
-  display: flex;
-  align-items: center;
-  flex-direction: column;
+.quiz-page * {
+  box-sizing: border-box;
 }
 
-.quiz > div:not([class]) {
-  display:none;
+.quiz-page .feds-topnav-wrapper {
+  padding: 0 20px;
 }
 
-.quiz-container {
-  display: flex;
-  align-items: center;
-  justify-content: flex-start;
-  flex-direction: column;
-  color: var(--color-white);
-  width: 100%;
+div[data-valign="middle"] {
```

Comment:
Should this not be scoped to the quiz logic?

Reply:
Completely unnecessary, as it turns out. It was only used in the background > div and it didn't actually apply. Just some old stuff that I had missed, good catch.